### PR TITLE
Consume both feeds in test

### PR DESCRIPTION
**Pull request by da-kami**
Minor changes on the test we already have. When creating another test I noticed that it reads nicer to just join on both channels. We might even pack the join into the `next` function at some point and return from both - I did not go that far yet. 

Note that this is also closer to what we do in the production code, because we now send the message with `do_send_async` which is what we do there.

### CODE REVIEW

**Review by DeliciousHair — APPROVED** [review]
LGTM!

File: daemon/tests/happy_path.rs [hunk 1]
```diff
@@ -153,11 +153,11 @@ impl Wallet {
 
 /// Maker Test Setup
 struct Maker {
-    cfd_actor_addr:
```
Comment (klochowicz) [inline daemon/tests/happy_path.rs:154]: if possible, let's use the version with `_addr` suffix in both Maker and Taker - then it's the same as in production code. and I guess it's good to reiterate it's just the address , not the actual actor being stored here.

**Review by klochowicz — APPROVED** [review]
indeed, it reads nicer! I like this pattern.

File: daemon/tests/happy_path.rs (hunk 1 above)
Comment (da-kami) [reply daemon/tests/happy_path.rs:154]: Actually, I do mind I think. Maybe we should change it in production code.

Reasoning:
The fact that this is an address and not the actor itself is an implementation detail for me.
When we talk about it we would also say that we "send a message to the actor" and not to the address of the actor.

**Comment by da-kami** [comment]
note: pushed a fixup that does not change much in the context of this test, but might change things with more complex tests.

File: daemon/tests/happy_path.rs (hunk 1 above)
Comment (da-kami) [reply daemon/tests/happy_path.rs:154]: If you insist I can change it, it's a minor thing for me :D

Comment (klochowicz) [reply daemon/tests/happy_path.rs:154]: I prefer, as it's nicer to read - or you can just drop this commit, so it stays as it was :p

Comment (klochowicz) [reply daemon/tests/happy_path.rs:154]: but it's not a blocker, so if you want to keep going, that's fine too.

Comment (da-kami) [reply daemon/tests/happy_path.rs:154]: 🤷‍♂️

**Pull request merged**